doc/api/process.md

```diff
@@ -1637,7 +1637,10 @@ Android operating system. However, Android support in Node.js
 
 ## process.ppid
 <!-- YAML
-added: v9.2.0
+added:
+  - v9.2.0
+  - v8.10.0
+  - v6.13.0
```

Comment:
I don't think this is the way we handle this, is it? /ping @vsemozhetbyt 

Unless I'm mistaken (which is a distinct possibility), we put the relevant information for this particular release line. So master branch and all branches later than v9.x should have v9.2.0, while the v8.x branch should have v8.10.0. Which it does! https://raw.githubusercontent.com/nodejs/node/v8.x/doc/api/process.md lhas this:

```markdown
## process.ppid
<!-- YAML
added: v8.10.0
-->
```

And similarly at https://nodejs.org/dist/latest-v8.x/docs/api/process.html#process_process_ppid.

I'm not sure our YAML processor will do the right thing with this...

Reply:
Oh, interesting, our YAML processor kinda mostly does handle this correctly. Here's what it ends up looking like:

<img width="232" alt="Screen Shot 2019-09-30 at 8 38 18 PM" src="https://user-images.githubusercontent.com/718899/65932378-4d352580-e3c2-11e9-87b0-65873d5625fd.png">

And I see that we've done this elsewhere in url.md and other places. Uh, ok, cool, I guess. Ignore me! 😀

Reply:
(It's definitely the exception though!)